org.gnome.Radio.json

```diff
@@ -0,0 +1,105 @@
+{
+  "app-id" : "org.gnome.Radio",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "43",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "gnome-radio",
+  "finish-args" : [
+     "--socket=x11",
+     "--share=ipc"
+  ],
+    "modules": [
+	{
+	    "name": "intltool",
```

Comment:
Please use intltool from the shared modules

Reply:
Fixed

Reply:
Fixed